### DISCUSSION TITLE
自增递归列表添加多条并配置默认值，编辑修改输入项时，其他条目会被同步修改

### DISCUSSION THREAD

**Discussion by yangshuwei** [alibaba/formily#1571] [Q&A 2.x]
[https://codesandbox.io/s/interesting-mclaren-mg3e1?file=/App.tsx](https://codesandbox.io/s/interesting-mclaren-mg3e1?file=/App.tsx)

**Comment by yangshuwei** [comment]
新添加两个条目，然后编辑第一个条目，这时第二个条目中的内容也被修改了，这个具体是什么原因呢？找了半天也没找到原因？

**Reply by janryWang** [reply]
codesandbox没复现

**Reply by yangshuwei** [reply]
arrayField push 对象类型的时候解构一下就可以了，相同的引用地址会导致我说的这个问题
